app/com/lucidchart/open/cashy/amazons3/S3Sync.scala

```diff
@@ -56,7 +56,7 @@ class S3Sync extends CloudfrontConfig {
   private def checkKrakenQuota() {
     val usageThreshold = configuration.getDouble("kraken.usageAlertThreshold").get
     val usage = KrakenClient.checkQuota()
-    if (!usage.isDefined || usage.get > usageThreshold) {
+    if (usage.isDefined && usage.get > usageThreshold) {
```

Comment:
usage will be None right after creating the Kraken account because their endpoint that returns quota data has a bunch of nulls in it for or some period of time.  So if it's None then we will just assume it's okay